### DISCUSSION TITLE
Pake 应用市场

### DISCUSSION THREAD

**Discussion by RayZhao1998** [tw93/Pake#283] [Ideas]
目前做了一个[网站](https://pakestore.vercel.app/),仅展示了 README 中的几个,后续计划:

- 支持搜索/分类
- 支持新增应用(PR方式)
- 支持网页触发打包

不知道大家还有没有什么想法

**Comment by tw93** [comment]
太太太感谢了，我们最近计划在做Pake的上层UI打包工具，里面有这个模块，有没有兴趣一起来弄，我的电报是 HiTw93，可以找我一下

**Reply by RayZhao1998** [reply]
已私
